src/elements/headerSubheader.jsx

```diff
@@ -0,0 +1,41 @@
+import ClassNames from 'classnames';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const propTypes = {
+    children: PropTypes.node.isRequired,
+    className: PropTypes.string,
+    id: PropTypes.string,
+    style: PropTypes.shape({}),
+};
+
+const defaultProps = {
+    className: undefined,
+    id: undefined,
+    style: {},
```

Comment:
Is it useful for `style` prop to default to an empty object versus `null` or `undefined`?  This is probably not a super important concern; I'm mostly just curious. ;-)

Reply:
You know, I have been thinking the same thing. I have going with empty object, because that’s what I started doing in the beginning of implementing the Airbnb Eslint rules. I think I would rather have undefined.